api/src/main/java/io/opencensus/stats/NoopStats.java

```diff
@@ -99,9 +110,31 @@ public void setState(StatsCollectionState state) {
     static final StatsRecorder INSTANCE = new NoopStatsRecorder();
 
     @Override
-    public void record(TagContext tags, MeasureMap measureValues) {
+    public StatsRecord newRecord() {
```

Comment:
Maybe recorder as opposed to record

Reply:
I like that name, except that the class containing this method is already called StatsRecorder.  I opened an issue for finding better names for StatsRecord and the related methods: #726